### PR TITLE
Chore: Make styles consistent in "Block Explorers" , "Connect"  and "…

### PR DESCRIPTION
**Pull request by stephanniegb**
…Faucet" modals

# Task name here
Make styles consistent in "Block Explorers" and "Faucet" modals
Fixes #relevant-issue-here
Issue #284
## Types of change

- [ ] Feature
- [ ] Bug
- [x] Enhancement

### CODE REVIEW

**Comment by Nadai2010** [comment]
Hi @stephanniegb the style of the connectwallet should not be touched, only the style of Faucet and Block Explorer should be adjusted, following the same consistency.

- The ConnecModal was fine, it was the reference to follow, only adjust if you think it is necessary the alignment, size but do not change the borders, background colors, or design:
Captura desde 2024-09-28 20-33-46
Captura desde 2024-09-28 20-38-41

---

- Adjust the same style that maintains connecting Wallet to Explorer and Faucet (borders and design):
imagen
Captura desde 2024-09-28 20-42-15

---

What do you think @metalboyrick ?

**Comment by metalboyrick** [comment]
Hey @stephanniegb , thanks for the hard work! Really appreciate your PR,

Yeah, I'm of the same thoughts as Nadai, I meant to say that the other styles should follow the on in "Connect Wallet". Apologize for the miscommunication!

Feel free to comment on this PR or the telegram support group should you need more info or clarity! Thank you!

**Comment by stephanniegb** [comment]
Okay @metalboyrick and @Nadai2010, I misunderstood. I increased the size of the connect modal to match the rest of the modals—should I change that?

**Comment by metalboyrick** [comment]
@stephanniegb I think for now you can leave the connect modal as is and apply the style changes to the two modals mentioned in the issue, thank you!

File: packages/nextjs/components/scaffold-stark/BlockExplorer.tsx [hunk 1]
```diff
@@ -4,6 +4,8 @@ import { Address as AddressType, mainnet } from "@starknet-react/chains";
 import { MagnifyingGlassIcon } from "@heroicons/react/24/outline";
```
Comment (metalboyrick) [inline packages/nextjs/components/scaffold-stark/BlockExplorer.tsx:4]: So far looks good ! Kindly check also the button in the faucet when the network is set to `devnet`. Would be great for it to have the same style as well (still in scope of this task).

**Review by metalboyrick — CHANGES_REQUESTED** [review]

**Review by Nadai2010 — APPROVED** [review]

File: packages/nextjs/components/scaffold-stark/BlockExplorer.tsx (hunk 1 above)
Comment (Nadai2010) [reply packages/nextjs/components/scaffold-stark/BlockExplorer.tsx:4]: Hi @stephanniegb, I agree with @metalboyrick. A review of the local faucet may be very helpful, but I really like how it turned out; it maintains a consistent style across the different components.

The light tone does not differentiate well among the various options; it lacks migrate the gradient border color.

image

Comment (stephanniegb) [reply packages/nextjs/components/scaffold-stark/BlockExplorer.tsx:4]: Okay, I will work on them

**Review by metalboyrick — APPROVED** [review]
Looks good to me!

**Pull request merged**